src/main/java/pl/edu/zut/mad/schedule/util/ModelMapper.kt

```diff
@@ -29,10 +30,14 @@ internal class ModelMapper {
                 val subjectWithCourseType = "$subject ($courseType)"
                 val teacherFullNameWithRoom = "${teacher?.academicTitle} ${teacher?.name} ${teacher?.surname} $room"
                 val isCancelled = reservationStatus.equals(CANCELED_LESSON_TEXT, true)
-                LessonUi(teacherFullNameWithRoom, subjectWithCourseType, isCancelled, timeRange ?: TimeRange())
+                val timeRangeUi = toUiTimeRange(timeRange ?: TimeRangeApi())
```

Comment:
Is this tested?

Reply:
Tests already exists, but they have to been fixed with  https://github.com/MADWI/schedule/pull/7/commits/2b383b96196b9ea5f327f9eceba3c208c6660c53 :)